source/frontend/src/utils/mapPropertyUtils.ts

```diff
@@ -87,3 +90,48 @@ export const getApiPropertyName = (property?: Api_Property): PropertyName => {
   };
   return getPropertyName(mapProperty);
 };
+
```

Comment:
centralizing these makes sense.

Reply:
This is what consumed me the most amount of time. We are not being totally thorough with the ts typing causing some issues where data is allegedly a type but internally is a different one